protected/humhub/modules/admin/grid/SpaceImageColumn.php

```diff
@@ -9,7 +9,6 @@
 namespace humhub\modules\admin\grid;
 
 use humhub\modules\space\widgets\Image as SpaceImage;
-use humhub\modules\space\models\Space;
```

Comment:
`$model` = Space class call

Reply:
Is this the best way to solve this problem? How do I know the relation between Space and $model?

Reply:
`getSpace()` is called
https://github.com/humhub/humhub/blob/d79fc2010c18cc2ccda0f05e0f28235dece058d4/protected/humhub/modules/admin/grid/SpaceImageColumn.php#L34-L37

Reply:
I am no expert in PHP. So please correct me if I am wrong.
$this refers to the class SpaceImageColumn. Or to ...? 
$model is variable with no reference to the use. How should PHP make a link between both?
So why do we need use humhub\modules\space\models\Space;?